astropy/units/format/ogip.py

```diff
@@ -19,6 +19,7 @@
 import keyword
 import math
 import os
+import copy
```

Comment:
The same code is in `fits.py` (which does have the `import copy` statement, and it just asks whether the scale of the unit is a power of 10. So, legit code. Though in the case of `ogip`, I must admit I cannot see any way the actual code ever is executed (since it is only ever used for deprecated units, the single one of which (Crab) is not a composite unit).

Anyway, bottom line for now: just add the import statement!